similarity/README.md

```diff
@@ -53,8 +51,25 @@ String s1="please turn on the monitor";
 double score=nesim.compare(s1,s2);
 ```
 
-To get the basic LLM similarity score, just set `usePhraseSim` and `useNER` as false in config file (default setting).
+To use this metric properly, you need to specify some configurations in the Config file.
+
+`wordMetric` is the  word comparator metric used in LLM. It can be chosen from "word2vec", "paragram", "esa", "glove", "wordnet" "phrase2vec" or "customized" (your own embedding file).
 
-To use `usePhraseSim` option, set it as true and use `phrase2vec` as `wordMetric`. The system can tokenized the sentence into phrase-based units and it will reformat the sentence. E.g. "please turn the light on" => "please turn-on the light".
+`usePhraseSim` option will automatically tokenized the sentence into phrase-based units when comparing sentences and notice it should be used with "phrase2vec". To use this option, set it as true and use `phrase2vec` as `wordMetric`. The system can identify the phrases and reformat the sentence. E.g. "please turn the light on" => "please turn-on the light". Notice: the phrase identification process depends on the generalized phrases dictionary we extracted from Wordnet (see `src/main/resources/phrases.txt`).
```

Comment:
Could you clarify this? 
"Notice: the phrase identification process depends on the generalized phrases dictionary we extracted from Wordnet". It is the first time "phrase identification" is mentioned in the text.